python/lsst/faro/utils/tex.py

```diff
@@ -450,7 +450,10 @@ def calculateTEx(data: List[CalibratedCatalog], config):
             (catalog["slot_PsfFlux_instFlux"] / catalog["slot_PsfFlux_instFluxErr"])
             > snrMin
         )
-        & (catalog["deblend_nChild"] == 0)
+        # Only use sources that are single sources
```

Comment:
Not particularly fan of inserting comments in between a Python statement. Could you move it right below `snrMin = 50` line above?